lib/cfg.h

```diff
@@ -61,6 +64,11 @@ struct _GlobalConfig
   /* hex-encoded syslog-ng major/minor, e.g. 0x0201 is syslog-ng 2.1 format */
   gint user_version;
 
+  /* config identifier specified by the user */
+  gint64 user_config_id;
+
+  guint8 config_hash[CONFIG_HASH_LENGTH];
```

Comment:
I would probably make this an allocated value so we don't need to include openssl into cfg.h

Reply:
Fixed, thanks.